cmd/dep/testdata/harness_tests/init/glide/case3/final/Gopkg.lock

```diff
@@ -14,8 +14,8 @@
   version = "v0.8.1"
 
 [solve-meta]
-  analyzer-name = "dep+import"
+  analyzer-name = "dep"
```

Comment:
Doh! I wonder why this didn't get picked up sooner? Thanks for fixing!

Reply:
👍